### DISCUSSION TITLE
Share folders of People

### DISCUSSION THREAD

**Discussion by Hainer1** [matiasdelellis/facerecognition#676] [Ideas]
Feature I would like to request:
I would like to be able to select one or more folders of people and share them within a group.

For example, a family use case. Each member of the family should be able to see the pictures of each selected person: mum, dad, child1, child2,...

Similar to Google Photos

Or is there a solution?

**Comment by ErreUno** [comment]
Hello!
Same request:
Family-photos are stored in an (admin) shared folder and cannot be found by members. Only "own" photos.
I tried the "handle_shared_files" option - but it seems not be the purpose.
regards EU
